src/Network/Receive/ServerType0.pm

```diff
@@ -3014,73 +2286,6 @@ sub top10 {
 	}
 }
 
-sub top10_alchemist_rank {
-	my ($self, $args) = @_;
-
-	my $textList = bytesToString(top10Listing($args));
-	message TF("============= ALCHEMIST RANK ================\n" .
-		"#    Name                             Points\n".
-		"%s" .
-		"=============================================\n", $textList), "list";
-}
-
-sub top10_blacksmith_rank {
```

Comment:
top10_blacksmith_rank was removed from ServerType0 and added to Receive, but it's still in Sakexe_0